src/components/NewsRoomCard.jsx

```diff
@@ -0,0 +1,58 @@
+import {
+  Card,
+  CardContent,
+  CardFooter,
+  Button
+} from "@baltimorecounty/dotgov-components";
+import React from "react";
+
+const NewsRoomCard = props => {
+  const { publishDate, title, author, articleSummary, thumbnail, url } = props;
+
+  const options = {
+    year: "numeric",
+    month: "long",
+    day: "numeric"
+  };
+
+  const published = new Date(publishDate).toLocaleDateString("en-US", options);
+
+  return (
+    <Card className="text-left">
+      <CardContent>
+        <div class="row">
+          <h3>{title}</h3>
+        </div>
+        <div className="d-flex col-12">
+          <div class="row">
+            <div id="dg_main-content">
+              <img
+                className="dg_image left"
+                src="//baltimorecountymd.gov/sebin/t/t/homepage-county-executive.jpg"
+                alt="County Executive Olzsewski delivers a speech."
```

Comment:
alt tag should also come from the api. We can add the property `thumbnailAltText`

Reply:
right now Im using the Category label (story, news release). Once we get the new property I can change it